app/src/main/java/com/example/perfectweatherallyear/repository/remoteData/WeatherData/WeatherRepositoryImp.kt

```diff
@@ -0,0 +1,12 @@
+package com.example.perfectweatherallyear.repository.remoteData.WeatherData
+
+import com.example.perfectweatherallyear.repository.remoteData.weatherapicom.model.ForecastResponse
+import retrofit2.Response
+import retrofit2.http.GET
+import retrofit2.http.Query
+
+interface WeatherRepositoryImp {
```

Comment:
It is not the implementation of the Repository. It is interface declaration of the weatherapicom API